_docs-sources/pipelines/overview/configuration.md

```diff
@@ -1,87 +1,360 @@
+
+*Supported Attributes:*
+
+- `plan_iam_role` (Required): The IAM role that Pipelines will assume when running Terragrunt plan commands.
+- `apply_iam_role` (Required): The IAM role that Pipelines will assume when running Terragrunt apply commands.
+- `region` (Optional): The AWS region that Pipelines will use when running Terragrunt commands. If not set, Pipelines will use the default region of `us-east-1`.
+- `session_duration` (Optional): The duration in seconds that the AWS session will be valid for. If not set, Pipelines will use the default session duration of `3600` seconds.
+
+### AWS Accounts Blocks
+
+AWS Accounts blocks are configurations used by `aws-oidc` [authentication](#authentication-blocks) blocks to have commonly re-used AWS account configurations codified and referenced by multiple authentication blocks.
+
+The label applied to an AWS Accounts block is the name of the AWS accounts block. This is a user-defined label for the collection of AWS accounts defined by the block, and must be globally unique.
+
+e.g.
+
+```hcl
+# .gruntwork/aws_accounts.hcl
+aws_accounts "all" {
+  path = "aws/accounts.yml"
+}
+```
+
+In this example, the `all` AWS accounts block is defined in a file named `aws_accounts.yml` within the `.gruntwork` directory. The `all` AWS accounts block references an external file located at `aws/accounts.yml` that contains the definitions of AWS accounts.
+
+DevOps Foundations customers may be familiar with the `accounts.yml` file as a file that is used by Account Factory to define the configurations of AWS accounts. In an effort to reduce the burden of migrating to the new configuration system, Pipelines uses the same schema for the `accounts.yml` file as Account Factory. This means that the `accounts.yml` file that is used by Account Factory can be used by the `aws_accounts` block without modification.
+
+The expected schema for the `accounts.yml` file is as follows:
+
+```yaml
+# required: Name of an account
+an_account:
+  # required: The AWS account ID
+  id: "an-aws-account-id"
+  # optional: The email address of the account owner
+  owner_email: "an-email-address"
+  # optional: Whether or not a VPC has been created in the account. Default is false.
+  vpc_created: true
+```
+
+Note that multiple AWS Accounts blocks can be defined, pointing to different `accounts.yml` files. This allows for the segmentation of AWS accounts into different YAML files for organizational purposes.
+
+The decision to leverage YAML files instead of HCL files for defining the configurations for AWS accounts was an intentional decision to increase the portability of these configurations for usage outside of Pipelines. Tools like [Terragrunt](https://github.com/gruntwork-io/terragrunt/) and [yq](https://github.com/mikefarah/yq) can be used to leverage these files, as they are more portable than HCL files.
```

Comment:
```suggestion
:::info
The decision to leverage YAML files instead of HCL files for defining the configurations for AWS accounts was an intentional decision to increase the portability of these configurations for usage outside of Pipelines. Tools like [Terragrunt](https://github.com/gruntwork-io/terragrunt/) and [yq](https://github.com/mikefarah/yq) can be used to leverage these files, as they are more portable than HCL files.
:::
```